gitsetup/pom.xml

```diff
@@ -16,20 +16,20 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>ddf.support</groupId>
-        <artifactId>support-pom</artifactId>
+        <groupId>codice</groupId>
+        <artifactId>codice-parent</artifactId>
         <version>2.3.13-SNAPSHOT</version>
     </parent>
 
-    <groupId>ddf.support</groupId>
+    <groupId>codice</groupId>
```

Comment:
groupId isn't really needed since it is the same as the parent. Not really part of your change though.